### PR TITLE
Update webidl2.js

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

File: resources/webidl2/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+set -ex
+
+if [ ! -d "webidl2.js" ]; then
+  git clone git@github.com:w3c/webidl2.js.git
```
Comment (foolip) [inline resources/webidl2/build.sh:4]:
```suggestion
  git clone https://github.com/w3c/webidl2.js.git
```

File: resources/webidl2/build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+fi
+cd webidl2.js
+npm run build-debug
```
Comment (foolip) [inline resources/webidl2/build.sh:7]: No `npm install` first?

File: resources/webidl2/build.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+VERSION=$(jq -r ".version" < package.json)
```
Comment (foolip) [inline resources/webidl2/build.sh:8]: Unless every commit is released, this version will be misleading, not the same thing that got published as that version on npm.

I would suggest either installing from npm (similar to the `@webref/idl` workflow) or just using the commit and not recording the version.

Comment (Ms2ger) [reply resources/webidl2/build.sh]: Fair enough. I went with not recording the version, since installing from npm seems to give minified code

**Review by foolip — APPROVED** [review]

**Pull request merged**